### PR TITLE
add lighthouse security docs

### PR DESCRIPTION
**Pull request by kaycebasques**

### CODE REVIEW

File: src/content/en/tools/lighthouse/http-redirects-to-https.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Site redirects HTTP traffic to HTTPS" Lighthouse audit.
+
+{# wf_updated_on: 2016-09-20 #}
+{# wf_published_on: 2016-09-20 #}
+
+# Site redirects HTTP traffic to HTTPS  {: .page-title }
```
Comment (jpmedley) [inline src/content/en/tools/lighthouse/http-redirects-to-https.md]: Title case, not sentence case.

File: src/content/en/tools/lighthouse/http-redirects-to-https.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+## How to pass the audit {: #how }
+
+1. Use canonical links in the `head` of your HTML to help search engines figure
+   out the best way to get to the page.
+
+       <link rel="canonical" href="https://example.com"/>
+
+2. Configure your server to redirect HTTP traffic to HTTPS. See your server's
+   documentation to figure out the best way to do this.
+
```
Comment (jpmedley) [inline src/content/en/tools/lighthouse/http-redirects-to-https.md]: Converting to HTTPS is considerably more complicated than you imply here. I believe Meggin is working with Jordan on an update of this for /web. Please link here when it's ready.

File: src/content/en/tools/lighthouse/https.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## Why the audit is important {: #why }
+
+All websites should be protected with HTTPS, even ones that don't handle
+sensitive data. HTTPS prevents intruders from tampering with or passively
+listening in on the communications between your site and your users.
+
+HTTPS is also a prerequisite for many new, powerful web platform features, such
+as taking pictures or recording audio.
+
```
Comment (jpmedley) [inline src/content/en/tools/lighthouse/https.md]: When we say why HTTPS is required, we frequently link to the policy. http://www.chromium.org/Home/chromium-security/prefer-secure-origins-for-powerful-new-features. We've done it on /web plenty of times.

File: src/content/en/tools/lighthouse/https.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+Migrate your site to HTTPS.
+
+Many hosting platforms, such as
+[Firebase](https://firebase.google.com/docs/hosting/) or [GitHub
+Pages](https://pages.github.com/), are secure by default.
+
+If you're running your own servers and need a cheap and easy way to generate
+certficates, check out [Let's Encrypt](https://letsencrypt.org/). For more help
+on enabling HTTPS on your servers, see the following set of docs: [Encrypting
+data in transit](/web/fundamentals/security/encrypt-in-transit/?hl=en)
```
Comment (jpmedley) [inline src/content/en/tools/lighthouse/https.md]: Please be careful with your periods and check throughout. I've found quite a few missing.

File: src/content/en/tools/lighthouse/http-redirects-to-https.md (hunk 2 above)
Comment (kaycebasques) [inline src/content/en/tools/lighthouse/http-redirects-to-https.md]: Migrating to HTTPS is very complicated, but this audit pertains specifically to redirecting HTTP to HTTPS, which is not that hard to set up.

File: src/content/en/tools/lighthouse/https.md (hunk 3 above)
Comment (kaycebasques) [inline src/content/en/tools/lighthouse/https.md]: I'll add a link to my canonical "why HTTPS" doc.

**Comment by PaulKinlan** [comment]
@petele outside of minor nits, these seems mergable.

**Comment by petele** [comment]
@kaycebasques - merges into lighthouse, once it's ready, please change so it merges into next-version and assign to me.

**Comment by kaycebasques** [comment]
@petele good to go

**Pull request merged**